composer.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "cslant/blog-core",
+  "description": "Send notification from Gitlab and Github events to Telegram",
```

Comment:
please update the description 
you can follow like this https://github.com/cslant/blog-api-package/blob/main/composer.json